ldms/man/ldmsd_controller.man

```diff
@@ -530,6 +530,15 @@ The updater name. If none is given, the statuses of all updaters are
 reported.
 .RE
 
+.SS Query the updater match list
+.BR updtr_match_list
+attr=<value>
+.RS
+.TP
+.BI [name " name"]
+.br
+The Updater name. If none is given, the match list of all updaters are reported.
```

Comment:
I think this is supposed to say "the match list of all sets ..." correct. We're not matching updaters here, were matching sets that an updater will be issuing ldms_xprt_update requests for?

Reply:
> I think this is supposed to say "the match list of all sets ..." correct. We're not matching updaters here, were matching sets that an updater will be issuing ldms_xprt_update requests for?

That was my understanding as well. When I wrote this command, it was meant to return updater sets that matched the given regex, not just the updaters that contain them.

Reply:
> I think this is supposed to say "the match list of all sets ..." correct. We're not matching updaters here, were matching sets that an updater will be issuing ldms_xprt_update requests for?

Yes, the match list is the list of regular expressions to match the sets the updater will be issuing ldms_xprt_update requests for.